index.html

```diff
@@ -0,0 +1,78 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <div class='navContainer'>
+        <div class="wrapper">
+            <nav>
+                <div class="logo">
+                    Ivan
+                </div>
+                <ul class="navItems">
+                    <li>
+                        <a href="#">Portfolio</a>
+                    </li>
+                    <li>
+                        <a href="#">About</a>
+                    </li>
+                    <li>
+                        <a href="#">Contact</a>
+                    </li>
+                    <li>
+                        <a class="navBtnMessage" href="#">
+                            <img class="messageBtn" src="images/mail.png" alt="">
+                        </a>
+                    </li>
+                </ul>
+                <ul class="navMenu">
+                    <li>
+                        <a class="navBtnMenu" href="#">
+                            <img class="menuBtn" src="images/menuUnion.png" alt="">
+                        </a>
+                    </li>
+                </ul>
+            </nav>   
+        </div>
+    </div>
+
+    <div class="headline-container">
+        <div class="wrapper">
+            <header>
+                <div class="headlineContent">
+                    <h1>Hey there. I'm Ivan.</h1>
+                    <span>I’m a Software Developer</span>
+                    <p>I can help you build a product, feature or website. Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.</p>
+                    <div class="headlineSocial">
+                        <a href="#">
+                            <img src="images/gitHub.png" alt="">
+                        </a>
+                        <a href="#">
+                            <img src="images/linkedin.png" alt="">
+                        </a>
+                        <a href="#">
+                            <img src="images/angelist.png" alt="">
+                        </a>
+                        <a href="#">
+                            <img src="images/twitter.png" alt="">
+                        </a>
+                        <a href="#">
+                            <img src="images/medium.png" alt="">
+                        </a>
+                        
+                    </div>
+                </div>
+
+                <div class="headlineImage">
+                    <img src="images/illustration.png" alt="">
+                </div>
+            </header>
+        </div>
+    </div>
```

Comment:
- [ ] This section should be inside `section` tag. Kindly replace `div` tag with `section` tag, as it represents a generic standalone section of the document, so use it. [Check this link](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/section) for more info.It represents a generic standalone section of the document, so use it. [Check this link](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/section) for more info.